cargo-shuttle/src/client.rs

```diff
@@ -62,6 +62,14 @@ pub(crate) async fn delete(
     Ok(())
 }
 
+pub(crate) async fn update_shuttle() -> Result<()> {
+    // run this command: "cargo install cargo-shuttle"
+
+    clap::command!("cargo", "install", "cargo-shuttle").run()?;
```

Comment:
This does not build for me! Looking at the [docs](https://docs.rs/clap/latest/clap/macro.command.html) I think this is supposed to be used to create a stub `Command` instance with things like author and version pre-filled from `Cargo.toml`.